docs/docs/reference/functions/getting-started.md

```diff
@@ -190,6 +190,16 @@ export default function FormPage() {
 }
 ```
 
+## Functions in plugins and themes
+
+Plugins and themes can ship functions! This is powerful as it lets you pair frontend code with backend code. E.g. imagine a plugin for an authorization service that includes a login component with a function to connect to the remote API.
+
+### Namespacing
+
+Plugin/theme Functions work exactly the same as normal functions except their routes must be created under the plugin's namespace e.g.: `${PLUGIN_ROOT}/src/api/{pluginName}/my-plugin-function.js`.
+
+Shadowing work with functions similar to how it does in general. You can shadow a plugin/theme Function by copying the file from its src tree into the site's src tree e.g. copy `node_modules/gatsby-plugin-cool/src/api/gatsby-plugin-cool/function.js` => `src/api/gatsby-plugin-cool/function.js`.
```

Comment:
> Shadowing with functions works similarly to how it works in general. 

Can you make this sentence more specific? Mainly the phrase "how it works". How what works? Shadowing? Functions? Namespaces?

Also added some context to the example

```suggestion
Shadowing with functions works similarly to how it works in general. You can shadow a plugin/theme Function by copying the file from the plugin/theme's `src` tree into your site's `src` tree. For example, to shadow the `/do-something` Function from the `gatsby-plugin-cool` plugin, you'd copy `node_modules/gatsby-plugin-cool/src/api/gatsby-plugin-cool/do-something.js` to `src/api/gatsby-plugin-cool/do-something.js`. From there, you can overwrite the implementation of the `/do-something` function as you normally would.
```